[DOCS-7434] Cloud SIEM Content Packs doc (#22999)
DataDog/documentation: content/en/security/cloud_siem/content_packs.md

---
title: Content Packs
kind: documentation
disable_toc: false
further_reading:
- link: "/security/cloud_siem/log_detection_rules"
  tag: "Documentation"
  text: "Create log detection rules"
- link: "security/cloud_siem/investigator"
  tag: "Documentation"
  text: "Learn more about the Investigator"
- link: "/security/cloud_siem/investigate_security_signals"
  tag: "Documentation"
  text: "Investigate security signals"
---

## Overview

> [Cloud SIEM Content Packs][1] provide out-of-the box content for key security integrations. The specific content depends on the integration and can include the following:

Reviewer: ```suggestion
[Cloud SIEM Content Packs][1] provide out-of-the box content for key security integrations. Depending on the integration, a Content Pack can include the following:
```

> - [Detection Rules][2] to provide comprehensive coverage of your environment

Reviewer 2: add:
- [Pipeline][x] to parse and normalize the logs

Author: @jnhunsberger which doc would we link to for "Pipeline"? Also, is this already available? I'm not seeing it listed for any of the content packs.

- An interactive dashboard with detailed insights into the state of logs and security signals for the Content Pack
- [Investigator][3], an interactive graphical interface for investigating suspicious activity by a user or resource
- [Workflow Automation][4] to automate actions and accelerate investigation and remediation of issues
- Configuration guides

{{< whatsnext desc="Content Packs are grouped into the following categories:" >}}

>     {{< nextlink href="/security/cloud_siem/content_packs#cloud-audit-content-packs" >}}Cloud Audit{{< /nextlink >}}

Reviewer: ```suggestion
    {{< nextlink href="/security/cloud_siem/content_packs#cloud-audit-content-packs" >}}<u>Cloud Audit</u>: AWS CloudTrail, Azure Security, GCP Audit Logs, Kubernetes Audit Logs{{< /nextlink >}}
```

so this is completely just an idea, but considering the relative length of this page (and in case more content packs get added) it might be nice to see an overview of available content packs in a somewhat-dense format at the top, so users can see them all at-a-glance and scan through. I think listing them up here would work—we use a vaguely similar format on the [kubernetes index page](https://docs.datadoghq.com/containers/kubernetes/). but again, absolutely just a suggestion!

Author: Ohhh this is a great idea. There definitely will be more content packs getting added. I added the others to the overview.

    {{< nextlink href="/security/cloud_siem/content_packs#authentication-content-packs" >}}<u>Authentication</u>: 1Password, Auth0, JumpCloud, Okta{{< /nextlink >}}
    {{< nextlink href="/security/cloud_siem/content_packs#collaboration-content-packs" >}}<u>Collaboration</u>: Google Workspace, Microsoft 365{{< /nextlink >}}
    {{< nextlink href="/security/cloud_siem/content_packs#network-content-packs" >}}<u>Network</u>: Cloudflare{{< /nextlink >}}
    {{< nextlink href="/security/cloud_siem/content_packs#cloud-developer-tools-content-packs" >}}<u>Cloud developer tools</u>: GitHub{{< /nextlink >}}
    {{< nextlink href="/security/cloud_siem/content_packs#endpoint-content-packs" >}}<u>Endpoint</u>: CrowdStrike{{< /nextlink >}}
{{< /whatsnext >}}

## Cloud Audit Content Packs

### AWS CloudTrail

Monitor the security and compliance levels of your AWS operations.

The [AWS CloudTrail Content Pack][5] includes:

> - Detection Rules

Reviewer 2: can we link to the available detection rules?
https://docs.datadoghq.com/security/default_rules/#cloudtrail

Author: added links to the detection rules for all content packs.
